rag/rag_practice/yes24_bestseller_crawling.py

```
@@ -0,0 +1,100 @@
+from urllib.request import urlopen
+from bs4 import BeautifulSoup as bs
+import pandas as pd
+import re
+
+BASE = 'http://www.yes24.com'
+LIST_URL = BASE + '/24/Category/BestSeller?CategoryNumber=001'
+
+# 목록 페이지 열기
+html = urlopen(LIST_URL)
```

Ruff (0.13.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)